nucliadb_protos/noderesources.proto

```diff
@@ -75,11 +75,8 @@ message ShardList {
 message EmptyResponse {}
 message EmptyQuery {}
 
-message VectorSentence {
-    repeated float vector = 1;
-}
 
-message ParagraphPosition {
+message Position {
```

Comment:
Is this a breaking change. Are we storing a protobuf with this message anywhere?

Reply:
AFAIK the message type name isn't part of the serialized binary in protobuf, so we should be fine:
https://protobuf.dev/programming-guides/encoding/#structure

According to the docs, we could even change the field name as long as we don't change the field type and the numeric field id.